### ISSUE TITLE
Is it too much to ask for a popup tooltip or some interaction when clicking on icon?

### ISSUE THREAD

**Issue by bair2** [vogu66/firefox-clip-url#2]
First off, I want to thank you for your efforts in restoring decaying Firefox to its semblance of its former self! Why did they remove this neat little functionality is beyond my understanding.

So I would like to ask you - if you remember how the Copy Page Link address button worked before - is it possible to breathe in at least some form of interactivity? 
I also noticed you have the code commented out in non-release file. Was the button supposed to blink when clicked?

Cheers,
Old time Firefox user.

**Comment by vogu66** [comment]
Well, you are quite welcome, it is appreciated. I would like to add, though that I do understand why Firefox developers remove features sometimes: it is better for everyone involved if the developers stay sane. In this particular instance (https://bugzilla.mozilla.org/show_bug.cgi?id=1699086), the problem was redundancy that needed to be reduced; redundancy is a developer's nightmare.

  On to the topic at hand, though. Initially, yes, I wanted some interactivity, and the thing that seemed easiest to implement was having the icon change color for a few second. It failed miserably though. The reason I didn't research further (yet) is that:
1) I cobbled the thing together in a few hours and I had/have exams coming up, can't dedicate too much time for this right now.
2) This was literally the second time in years that I touched some Javascript, and the second time ever that I did anything useful with it, so doing anything takes time.

  That said, I do plan to spend a bit more time on this, when I get the time, which should hopefully not be due too long. But it's not like I particularly want to, it's just that I'd like the add-on to be complete. So, if you (or anyone reading this) want to try your hand at adding interactive-ness, go ahead. As long as you don't make it play a song or have a dancing monkey popping up on the screen, obviously, I'd like it to stay discrete and usable on a ten-year-old machine.
